docs/osquery/osquery.asciidoc

```diff
@@ -0,0 +1,328 @@
+[chapter]
+[role="xpack"]
+[[osquery]]
+= Osquery
+
+*Osquery* enables you to search, analyze, and visualize Osquery data ingested in {es}.
+
+You can:
+
+ * Run live queries with one or more agents
+ * Schedule queries to capture changes to OS state over time
+ * View a history of past queries and their results
+
+For example, use *Osquery* to centralize security analytics and
+contextualize OSquery results against other event data, anomalies, and threats,
+and then use that context to improve host visibility, analytical power, and monitoring.
+
+[role="screenshot"]
+image::images/live-query-history.png[A table showing query name, agents, create time, and who ran the query]
+
+
+[float]
+== Requirements
+
+* You must have the
+https://www.elastic.co/guide/en/beats/filebeat/current/filebeat-module-osquery.html[Osquery Manager integration].
+This integration supports x86_64 bit Windows, Darwin, Arm64, and Linux platforms.
+* Your system has an agent in Fleet.
+* Osquery Manager has one or more policies. For more information, refer to the Fleet documentation.
+* You have the xxxx permission
+
+[float]
+[[osquery-open]]
+== Open Osquery
+
+Once you add the Osquery Manager integration to an agent policy, open *Osquery*
+and begin your exploring your data.
+
+. Open the main menu, and then click *Osquery*.
+. View the queries that have run.
+. To replay a query, click image:images/play-icon.png[Right-pointing triangle].
+. To view the query <<osquery-results,results>> and <<osquery-status,status>>,
+click image:images/table-icon.png[Table icon].
++
+[role="screenshot"]
+image::images/live-query-check-results.png[Results of OSquery]
+
+
+[float]
+[[osquery-run-query]]
+==  Run live queries
+
+Run a query against one or more agents or policies
+and view the results. This is useful for testing your query and
+troubleshooting problems.
+
+. In the *Live queries* view, click **New live query**.
+. Select one or more agents or groups to query. Start typing in the search field,
+and you'll get suggestions for agents by name, ID, platform, or policy.
+. Enter a query or select a query from your saved queries.
++
+[role="screenshot"]
+image::images/enter-query.png[Select saved query dropdown name showing query name and description]
+. Click **Submit**.
+. Review the results in a table, or navigate to *Discover* to dive deeper into the response
+or to *Lens* to build visualizations.
+. To view more information about the request, for example, if there are any failures, open the *Status* tab.
+. When your query captures the data you want, click *Save for later* and define the ID,
+description, and other
+<<osquery-manage-query,details>>.
+
+
+[float]
+[[osquery-schedule-query]]
+== Schedule queries
```

Comment:
Can we add a section about mapping osquery results for scheduled queries to ECS? This is a new feature in 7.15. I’m not sure if it should be a standalone section or included in the existing `Schedule queries` section. 
I’ve written a draft as a standalone section, but I’ll leave it to you @gchaps to decide whether that actually makes sense :)

> === Map osquery fields to ECS fields for scheduled query groups
> When scheduling queries, you can optionally map query results to ECS fields, so that you can standardize your osquery data for use across detections, machine learning, and any other areas that rely on ECS-compliant data. When you map an osquery field to an ECS field, the query results include both the original `osquery.<field>` and the mapped ECS field. For example, if you update a query to map `osquery.name` to `user.name` in ECS, the query results will include both fields.
> 
> . When editing a scheduled query group, click the edit icon for the query you want to map.
> 
> . In the **ECS mapping** section, select which osquery result fields you want to map to ECS fields. The fields available in the **Osquery.results** column are based on the SQL query entered and only include fields that will be returned by the query. 
> 
> . When mapping fields: 
> .. To add a new row for additional fields to map. click the plus icon.
> 
> .. To remove any mapped rows, click the Trash icon.
> 
> . Click Save to save changes to the query.
> 
> . Click Save query to save changes to the group.
>

Reply:
Resolving this comment - this has been added in a new section titled `Map Osquery fields to ECS fields`